network/protocol.go

```diff
@@ -32,6 +32,18 @@ import (
 	"github.com/ethersphere/swarm/state"
 )
 
+var (
+	capabilitiesFlagRetrieve      = 0
+	capabilitiesFlagPush          = 1
```

Comment:
Base-stuff first nibble, relay-stuff second nibble, sync highest bit because it seems to have the biggest impact on behavior. I'm not in love with the numbers, just seemed logical somehow.

Reply:
> Also these should be exported no?

Not sure yet. Let's see when we have API and have some client code calling it. I prefer the approach don't export until something is shown to need it?